src/modules/vendor/components/programs/cards/templates/FillTheGaps.vue

```diff
@@ -109,6 +109,7 @@ export default {
     },
     async updateAnswer (index) {
       try {
+        this.card.falsyGapAnswers[index] = this.card.falsyGapAnswers[index].trim();
```

Comment:
plutot a mettre dans un `map` apres le `filter` de `this.card.falsyGapAnswers.filter(a => !!a)` ce serait plus propre
et tu pourrais meme extraire ce code dasns une methode `formatFalsyGapAnswerPayload`